browser/BUILD.gn

```diff
@@ -113,6 +113,7 @@ source_set("browser") {
     "//brave/components/resources:brave_components_resources_grit",
     "//brave/browser/resources:brave_extension_grit",
     "//chrome/browser",
+    "//brave/components/brave_sync",
```

Comment:
why is this dep here? It doesn't appear to be used in brave_content_browser_client

Reply:
it looks like it should be a dep for `brave/browser:browser_process` instead